utils/build-script-impl

```diff
@@ -1281,6 +1281,36 @@ function cmake_config_opt() {
     fi
 }
 
+function copy_embedded_compiler_rt_builtins_from_darwin_host_toolchain() {
+    local clang_dest_dir=$1
```

Comment:
Does this need quotes on the `$1`? I haven't ever used `local` before so I don't know if it's special.

Reply:
local just indicates that the variable is scoped to the function.  The assignment should be fine without quoting because the value is being assigned not a literal.  The positional argument quoting with {} is less important because there are only 9 (1-9) so you can not have more than a single character.

Reply:
And this form still works if the value contains whitespace?

Reply:
Thanks for catching this, it will not -- addressed in 0ca06c2